### ISSUE TITLE
Improve nested exception reporting

### ISSUE THREAD

**Issue by leoquijano** [Nayjest/Grids#122]
While debugging an issue with a EloquentDataProvider, I ran across a small thing that can be improved to help people figure out where errors are. 

Let's take for example the following:

```php
$field = new FieldConfig();
$field->setName('complex_foo');
$field->setLabel("Complex Column");
$field->setSortable(false);
```

And then this in the associated model:

```php
public function getComplexFooAttribute() {
  ...
  return $this->bar->name;
}
```

Where "bar" is a ```BelongsTo``` relationship. Now, if I make a mistake and ```$this->bar``` is not defined (and I don't check for it), Nayjest Grid will only tell me this:

```
"Can't read 'complex_foo' property from DataRow"
```

But it won't tell me the underlying cause of the issue. Granted, in this simple example, it may be very simple to figure out. But in more complex getters, it's harder to get.

I suggest adding a nested Exception in ```Nayjest\Grids\ObjectDataRow``` to help people debug, while providing the additional information that Nayjest Grids gives:

```php
protected function extractCellValue($fieldName)
{
  // ...
  } else {
    try {
      return $this->src->{$fieldName};
    } catch(Exception $e) {
      throw new RuntimeException(
        "Can't read '$fieldName' property from DataRow", 0, $e
      );
    }
  }
}
```

Notice the usage of the ```$e``` argument for ```RuntimeException```. That will allow programmers to see the nested exception. (In this case a "Trying to get a property of non-object" error, along with a line number).

There are probably other areas where this can be added too. And you might want to use code numbers as well, instead of just 0.

**Comment by Nayjest** [comment]
Hi Leonardo!
Thanks for idea, nice improvement, I will add nested exception.

Also, try [vew-components/grids](https://github.com/view-components/grids) with [eloquent-data-processing](https://github.com/view-components/eloquent-data-processing), new grids is almost ready for release, I already use it on production. See usage examples: 
https://github.com/view-components/grids/blob/master/tests/webapp/Controller.php

**Comment by Nayjest** [comment]
Released in [v1.1.4](https://github.com/Nayjest/Grids/releases/tag/v1.1.4)
